src/Tizen.Wearable.CircularUI.Forms/IRotaryService.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;

```

Comment:
remove unused using

Reply:
Thanks for finding it out. It's fixed now.